setup.py

```diff
@@ -569,15 +573,31 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     include = data.get("include", [])
     include.append(numpy.get_include())
 
+    extra_comp_args = extra_compile_args.copy()
+    comp_macros = data.get("macros", macros)
+    undef_macros = []
+
+    if is_platform_zos():
+        language = data.get("language", None)
+        if language == "c++":
+            compiler = os.environ.get("CXX", "/bin/xlc++")
```

Comment:
Why we do assume that if `CXX` is not in env, the compiler is xlc++?

Reply:
On z/OS, xlc/xlc++ is the default compiler that is provided, so most users will have it.